src/components/views/passwordReset/NewPassword.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { applyReset } from '../../../redux/actions/passwordResetActions';
+import passwordValidator from '../../../utils/passwordValidator';
+
+const ResetPassword = ({
+  loading, success, message, resetPassword, match: { params },
+}) => {
+  const [formData, setFormData] = useState({
+    password: '',
+    confirmPassword: '',
+  });
+
+  const { password, confirmPassword } = formData;
+  const onChange = e => setFormData({ ...formData, [e.target.name]: e.target.value });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const valid = passwordValidator(password, confirmPassword);
+    if (valid === true) resetPassword(password, confirmPassword, params.token);
+  };
+
+  const error = !success && message;
+  return (
+    <div className="passwd-container">
+      <Link to="/">
+        <span className="brand-logo logo">
+          <img src="https://bit.ly/2lYgVI9" alt="logo" />
```

Comment:
I suggested making the link to the logo an `env` variable so that if it ever changes, we won't be making the update in several places

Reply:
I tried doing that. After installing importing dotenv I got an error saying `can't resolve fs in node_modules/dotenv/lib`. I had to push the branch so you can review the current changes while I look for a way around it.